packages/material-ui-codemod/README.md

```diff
@@ -0,0 +1,41 @@
+# Material-UI codemod
+
+This repository contains a collection of codemod scripts based for use with
+[JSCodeshift](https://github.com/facebook/jscodeshift) that help update Material-UI
+APIs.
+
+## Setup & Run
+
+- `npm install -g jscodeshift`
+- `npm install material-ui-codemod`
+- `jscodeshift -t <codemod-script> <file>`
```

Comment:
Signature according to https://github.com/facebook/jscodeshift#usage-cli is actually `<path>`, you can do whole directories not just individual files, which `<file>` might lead one to think be the case.